src/memote/suite/tests/test_consistency.py

```diff
@@ -153,11 +154,16 @@ def test_detect_energy_generating_cycles(model, met):
 
     """
     ann = test_detect_energy_generating_cycles.annotation
-    if met not in model.metabolites:
+    # Test if the metabolite is present in the model.
+    main_comp = helpers.find_compartment_id_in_model(model, "c")
```

Comment:
There is probably a more general way to find the compartment, but "c" should work for most models. I must admit it's been a while for me to think about.

Reply:
I agree. I did it this way, since it's what the `detect_energy_generating_cycles` function does, so that we know anything found here will be found in the function:
https://github.com/opencobra/memote/blob/4c08f46849628507ac8c4de56b5d6ec745feed0b/src/memote/support/consistency.py#L356